_targets.R

```diff
@@ -17,70 +15,84 @@ options(tidyverse.quiet = TRUE)
 ###parameters  
 NWIS_parameter <- '00060'
 startDate<-as.Date("1900-10-01") 
-endDate<-as.Date("2020-09-30")
+endDate<-as.Date("2020-9-30")
 ##water year or calendar year.  I assume we are doing this on water year but
 ##Eflow stats can do either one and it needs to be specified 
 yearType<-"water"
+##number of complete years we require for a site to be
+complete_years <- 10
+#precentile for flood threshold in Eflowstats.  0.6 is the default
+perc<-0.6
 
-##note -we will need a screening function to get a list of potential 
-##gages from gagesII to make the p1_sites_list target.For now, I'm just using a few sites.
-p1_sites_list<- c("06746095","06614800", "09035800","07083000","07086500")
+###gages2.1 ref site list - not sure how to get this right from sharepoint, so the
+##filepath is currently to my onedrive.
+gagesii_path<-"C:/Users/slevin/OneDrive - DOI/FWA_bridgeScour/Data/Gages2.1_RefSiteList.xlsx"
```

Comment:
That's fine. I don't know if it would be able to access directly from the SharePoint http address because of permissions. Can you add a note to the Readme about GAUGES 2.1 being stored in a private repo until the release is public? And if it's not already an issue, can you make an issue to remind us to switch to the public version when it's available?

Reply:
ok - created issue for this.